ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs

```diff
@@ -54,14 +54,17 @@ public static bool IsOnDispatcher()
             return CurrentDispatcher.CheckAccess();
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
+#pragma warning disable AvoidAsyncVoid
         public static async void RunOnDispatcher(Action action)
+#pragma warning restore AvoidAsyncVoid
         {
             AssertDispatcherSet();
 
             await CurrentDispatcher.InvokeAsync(action).Task.ConfigureAwait(false);
```

Comment:
@kevinvangelder It turns unaccessible from other classes when I change the return type and  add suffix. I'll leave it for now and check if there are same issues on UWP.